src/Krank/Checkers/IssueTracker.hs

```diff
@@ -102,31 +101,36 @@ extractIssues filePath toCheck = case parse (findAllCap patterns) filePath toChe
   Right res -> map snd $ rights res
   where
     patterns = localized $ choice [
-      githubRE
-      -- gitlabRE -- TODO: enable gitlab again
+      githubParser
+      , gitlabParser
       ]
 
 -- Supports only github for the moment
 issueUrl :: GitIssue
          -> Req.Url 'Req.Https
 issueUrl issue = case server issue of
   Github -> Req.https "api.github.com" Req./: "repos" Req./: owner issue Req./: repo issue Req./: "issues" Req./: (pack . show $ issueNum issue)
-  -- Gitlab -> Req.https "google.com"
+  Gitlab -> Req.https "gitlab.com" Req./: "api" Req./: "v4" Req./: "projects" Req./: ([fmt|{owner issue}/{repo issue}|]) Req./: "issues" Req./: (pack . show $ issueNum issue)
 
 -- try Issue can fail, on non-2xx HTTP response
 tryRestIssue :: Req.Url 'Req.Https
              -> ReaderT KrankConfig IO Value
 tryRestIssue url = do
   mGithubKey <- githubKey <$> ask
+  mGitlabKey <- gitlabKey <$> ask
   let
     authHeaders = case mGithubKey of
       Just (GithubKey token) -> Req.oAuth2Token (BSU.fromString token)
       Nothing -> mempty
+    gitlabHeader = case mGitlabKey of
+      Just (GitlabKey token) -> Req.header "PRIVATE-TOKEN" (BSU.fromString token)
+      Nothing -> mempty
 
   Req.runReq Req.defaultHttpConfig $ do
     r <- Req.req Req.GET url Req.NoReqBody Req.jsonResponse (
       Req.header "User-Agent" "krank"
-      <> authHeaders)
+      <> authHeaders
```

Comment:
Is it correct to send both tokens to the same API? Is this secure? I'm concerned about leaking token to another API.

Reply:
True, in the case where you would have both a githubkey and a gitlabkey, then this would be a concern.
Did not think about it as I figured most case would be in one repo or another but that's not true.

That means we'll need to make it a bit more complicated though.

Reply:
done